libs/java/server_common/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
I had to add this file (it is also located in the same directory in zts). It is used to tell mockito that it can mock final methods (without it some tests failed).